src/__generated__/ArtistForSaleArtworksGridQuery.graphql.ts

```diff
@@ -418,7 +412,13 @@ return {
                             "args": null,
                             "storageKey": null
                           },
-                          v2,
+                          {
+                            "kind": "ScalarField",
+                            "alias": null,
+                            "name": "is_biddable",
+                            "args": null,
+                            "storageKey": null
+                          },
                           {
                             "kind": "ScalarField",
```

Comment:
o.0

Wonder what's going on here /cc @alloy

Reply:
What should I be looking at?

Reply:
this PR is full of these diffs doing nothing, which feels like the JSON is non-deterministic

Reply:
It absolutely _should_ be deterministic. It feels like maybe field selections may have changed order at some point or something (and maybe not checked in)?